Client/src/ArmoniK.DevelopmentKit.Client.Common/Submitter/BaseClientSubmitter.cs

```diff
@@ -90,46 +90,6 @@ public BaseClientSubmitter(ILoggerFactory loggerFactory)
     /// </summary>
     protected Api.gRPC.V1.Submitter.Submitter.SubmitterClient ControlPlaneService { get; set; }
 
-
```

Comment:
Wasn't this code used?

Reply:
It was but it is not correct anymore with the fact that the taskId and the resultId are not the same.